src/lib/fragments/AgGrid.react.js

```diff
@@ -1010,6 +1017,38 @@ export default class DashAgGrid extends Component {
         }
     }
 
+    scrollTo(reset = false) {
+        const {gridApi} = this.state;
+        const {scrollTo, setProps} = this.props;
+        if (!gridApi) {
+            return;
+        }
+        if (scrollTo.rowIndex || scrollTo.rowIndex === 0) {
+            const rowPosition =
+                scrollTo.rowPosition === null ? 'top' : scrollTo.rowPosition;
+            gridApi.ensureIndexVisible(scrollTo.rowIndex, rowPosition);
+        }
+        if (scrollTo.rowId) {
+            const rowPosition =
+                scrollTo.rowPosition === null ? 'top' : scrollTo.rowPosition;
+
+            const node = gridApi.getRowNode(scrollTo.rowId);
+            gridApi.ensureNodeVisible(node, rowPosition);
+        }
```

Comment:
add `node` or `data` here, also, I'd recommend an else if instead of ifs.

For example, say a dev passed three different rows for the grid to scroll to, `rowIndex`, `rowId` and `data`, this would the finally end up on data, but could cause some interesting glitches.

something like: `if rowIndex else if rowId else if node/data`

Bring the `rowPosition` declaration outside of the ifs, since this isnt going to change.

I dont believe that you need the `=== null`, this seems redundant.